spec/controllers/items_controller_spec.rb

```diff
@@ -223,15 +223,55 @@
   describe '#rights' do
     before do
       allow(Dor::Services::Client).to receive(:object).with(pid).and_return(object_client)
-      allow(Dor).to receive(:find).with(apo.pid).and_return(apo)
     end
 
     let(:object_client) { instance_double(Dor::Services::Client::Object, find: cocina) }
     let(:cocina) { instance_double(Cocina::Models::DRO, version: 1) }
 
-    it 'renders the modal with rights selector' do
-      get :rights, params: { id: pid }
-      expect(response).to have_http_status(:ok)
+    context 'without an assigned apo' do
+      before do
+        get :rights, params: { id: pid }
+      end
+
+      let(:apo) { nil }
```

Comment:
Hmm, where is this variable being used?

Reply:
In `@rights_list = rights_list_for_apo(@apo)`

Reply:
I mean in that block of tests. I don't see references to `apo` until the next `context` block.

Reply:
OK, I think I see now. Here? https://github.com/sul-dlss/argo/blob/amcollie_446/spec/controllers/items_controller_spec.rb#L20

Reply:
That's the one